### PR TITLE
ffmpeg: update to 4.4.1

### PR DESCRIPTION
**Pull request by vcyzteen**
<p align="center">

# <p align="center">`update ffmpeg package`</p>

### CODE REVIEW

File: packages/ffmpeg/build.sh [hunk 1]
```diff
@@ -3,10 +3,10 @@ TERMUX_PKG_DESCRIPTION="Tools and libraries to manipulate a wide range of multim
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # NOTE: mpv has to be rebuilt and version bumped after updating ffmpeg.
-TERMUX_PKG_VERSION=4.4
+TERMUX_PKG_VERSION=4.4.1
 TERMUX_PKG_REVISION=3
```
Comment (buttaface) [inline packages/ffmpeg/build.sh:7]: Remove this revision line.

Comment (vcyzteen) [reply packages/ffmpeg/build.sh:7]: Ok sir. wait let me remove the revision message

**Comment by buttaface** [comment]
Looks like it almost works but will require a change to `nm`.

**Comment by vcyzteen** [comment]
> Looks like it almost works but will require a change to `nm`.

oh. 

> Looks like it almost works but will require a change to `nm`.

oh. btw sorry sir what is nm?

**Comment by buttaface** [comment]
It is a tool used by the Android NDK to check what ELF symbols are in a binary. It looks like ffmpeg expects it to have a different name, while the recent update to NDK 23 switched to using `llvm-nm`. We'll need to patch that to get this to build.

**Comment by tomty89** [comment]
Probably `--nm="$NM"` will do? https://github.com/FFmpeg/FFmpeg/blob/n4.4.1/configure#L372

**Comment by vcyzteen** [comment]
> Probably `--nm="$NM"` will do? https://github.com/FFmpeg/FFmpeg/blob/n4.4.1/configure#L372

I don't know sir. maybe is need to try?

**Comment by vcyzteen** [comment]
> It is a tool used by the Android NDK to check what ELF symbols are in a binary. It looks like ffmpeg expects it to have a different name, while the recent update to NDK 23 switched to using `llvm-nm`. We'll need to patch that to get this to build.

thanks for the explanation sir. I closed this problem first because I don't really understand about llvm-nm and patches. maybe i will come back another day

**Pull request closed without merging**